### ISSUE TITLE
@SpireInitializer?

### ISSUE THREAD

**Issue by LilithChamber** [daviscook477/BaseMod#109]
I'm not too sure what that means. I never used Eclipse, and the wiki just jumps to that step without any introduction. I'm not sure where to find SpireInitializer, or if that's something that I should create myself. Even so, if I have to create myself, where do I put it? Do I put it inside one of the jars, or in the "src/main/java"?

**Comment by kiooeht** [comment]
`@SpireInitializer` comes from ModTheSpire (https://github.com/kiooeht/ModTheSpire/wiki/SpireInitializer), which should of been added as a dependency if you followed all the previous steps.

**Comment by LilithChamber** [comment]
Mod the spire is in the dependecies, yes. But I can't find anything called "@SpireInitializer" anywhere, and if I open the mod the spire.jar, it just shows a huge list of stuff. Besides that, after that there is a topic named "Mod Logic" that lists Initializer and more stuff. Will the entire mod be entire one singular JAR, or what? Sorry, I'm quite a noob at this, I'm not sure where I should "Write" my code. 
![image](https://user-images.githubusercontent.com/42228618/43997364-329c00a4-9daf-11e8-8494-92500f1f96bd.png)

**Comment by DenizGelion** [comment]
Yeah, I did some research on how to mod StS and stumbled upon this, I followed the instructions for a bit, but then I have no idea on how to continue. I downloaded some character mods to look at how they're structured and I'm still not quite sure how and where I am supposed to do stuff? It would greatly help to see examples for like REALLY small mods. Like an extra character. A copy of the ironclad. Where do I do what? Idk what prerequisites you are expecting and I don't know if every mod author just magically knows what to do, but I just wanted to make a little birthday mod :(
